Simplenote/MagicLinkRequestedView.swift

```diff
@@ -0,0 +1,80 @@
+import Foundation
+import SwiftUI
+import Gridicons
+
+
+// MARK: - MagicLinkRequestedView
+//
+struct MagicLinkRequestedView: View {
```

Comment:
Currently almost all of the SwiftUI in SNiOS is done with UIKit.  The only swiftUI we are using is in the widgets, where it is required.  Do we want to start mixing UIKit and SwiftUI?

Reply:
Discussed over Slack, but leaving a note here, for the record!!

IMHO SwiftUI is all good, as long as it doesn't cause problems. That'd be: perhaps we should just try avoid blending SwiftUI + UIKit in the same UI, because of the (known) layout loops.

TY for bringing this up!!